inference-engine/src/mkldnn_plugin/nodes/common/tile_broadcast_utils.cpp

```diff
@@ -211,23 +211,62 @@ bool TileBroadcastCommon::prepareOptimizedParams(const MKLDNNNode *node, VectorD
     return true;
 }
 
+void TileBroadcastCommon::broadcastScalar(const char *srcData, char *dstData, size_t elt_cnt, size_t data_size) {
```

Comment:
The algo inside is a little bit unclear. Could you please add brief explanation?

Reply:
Fixed. Thanks Dmitry!